pages/linux/cointop.md

```diff
@@ -0,0 +1,24 @@
+# cointop
+
+> Track and monitor crypto in the terminal.
```

Comment:
```suggestion
> Track and monitor cryptocurrencies in the terminal.
```

I assume you mean cryptocurrencies and not cryptography? ;)

Reply:
Yes, thank you